### PR TITLE
[IMP] website: introduce `s_shape_image` snippet

### PR DESCRIPTION
**Pull request by chgo-odoo**
This PR introduces the new `s_shape_image` snippet.

- requires https://github.com/odoo/design-themes/pull/885

image

task-4094366
Part of task-4077427

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/views/snippets/s_shape_image.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+<template id="s_shape_image" name="Shape image">
+    <section class="s_shape_image pt32 pb32">
+        <div class="container">
+            <div class="row align-items-center">
+                <div class="col-lg-5 pt16 pb16">
+                    <img class="img img-fluid" data-shape="web_editor/composite/composite_double_pill" data-file-name="s_shape_image_default_image.jpg" data-original-mimetype="image/jpeg/" src="/web_editor/image_shape/website.s_shape_image_default_image/web_editor/composite/composite_double_pill.svg"/>
+                </div>
+                <div class="col-lg-6 offset-lg-1 pt16 pb16">
+                    <h3>About our product line</h3>
```
Comment (Brieuc-brd) [inline addons/website/views/snippets/s_shape_image.xml:12]:
```suggestion
                    <h2 class="h3-fs">About our product line</h2>
```

**Review by Brieuc-brd — CHANGES_REQUESTED** [review]
Thanks @chgo-odoo 💪 
I've just left a small comment below, good job :)

Also, some extra notes :
- About your commit description : 
> This commit introduces a new inner snippet named `s_shape_image`.

Your snippet doesn't seems to be an inner snippet :)

- About the text, since we've decided to add more meaningful content, I think we could improve it in this snippet (as you did in DT 😉 ).

Could you check ?

**Comment by Brieuc-brd** [comment]
@chgo-odoo 
Since the purpose is the same as `s_text_image`, I think it would make sense to move this snippet next to `s_text_image` in the list for the modal, what do you think?

**Review by Brieuc-brd — APPROVED** [review]
LGTM, good job @chgo-odoo 👍 

I just moved the snippet closer than `s_image_text` in the modal list.

**Comment by sobo-odoo** [comment]
@chgo-odoo I will adapt the PRs since the changes asked are small, don't change anything on it from now :wink:

**Comment by sobo-odoo** [comment]
Note:
- Rebased + conflicts ([diff](https://github.com/odoo/odoo/compare/630f9290938f7f5989e52d84311fc992b55955a4..109f44f3096d8b2d31b016a061491951821dbce0)) 
- Fixed the jpeg issue (see [diff](https://github.com/odoo/odoo/compare/109f44f3096d8b2d31b016a061491951821dbce0..2e2e922598bdda6eaab9d1f54692dc3e405cd621)) :slightly_smiling_face:

**Review by sobo-odoo — APPROVED** [review]
It's okay with me :slightly_smiling_face:
Will move it to the "super PR" haha

**Comment by sobo-odoo** [comment]
Commit moved to
- https://github.com/odoo/odoo/pull/180736

**Pull request closed without merging**